app-war/pom.xml

```diff
@@ -30,6 +30,8 @@
 							<Ignore-Scanning-Archives>asm-all-repackaged-2.5.0.jar</Ignore-Scanning-Archives>
 						</manifestEntries>
 					</archive>
+					<!-- warName becomes the context of the application, it avoids container specific configuration -->
+					<warName>${artifactId}</warName>
```

Comment:
What if we make this a property that defaults to the artifact+version and projects or (pipeline) can override it if they need to change the name.